tests/algorithms/test_algorithms.py

```diff
@@ -11,7 +11,7 @@
                                Momentum, AdaDelta, BasicRMSProp, RMSProp, Adam,
                                AdaGrad, RemoveNotFinite, Restrict,
                                UpdatesAlgorithm)
-from blocks.utils import shared_floatx, shared_floatx_zeros
+from blocks.utils.theano_utils import shared_floatx, shared_floatx_zeros
```

Comment:
As far as I understood you added a hack to `blocks/utils.py` so that one can still write `from blocks.utils import stuff`. If this is correct, why are you accessing `theano_utils` directly here?